data/human/free worlds 0 prologue.txt

```diff
@@ -1068,6 +1068,31 @@ mission "FW Bounty 1"
 
 
 
+ship "Behemoth" "Behemoth (Moonless Night)"
+	sprite "ship/behemoth moonless"
+	thumbnail "thumbnail/behemoth moonless"
+	add attributes
+		"shields" -300
+		"hull" 200
```

Comment:
I'd add 500 hull otherwise these changes just are a nerf since hull is just shield that doesnt regen and doesnt protect from specisl damages and takes extra dmg from lasers
But then idk much about balance